.github/workflows/daily-scheduled-ci.yml

```diff
@@ -0,0 +1,68 @@
+name: Daily GeNet CI Build
+
+on:
+  schedule:
+    - cron: '37 14 * * 1-5'
```

Comment:
2.37 p.m UTC. If you kick off your scheduled builds on the hour like most people, you will sometimes find there is no runner available and your job waits in a queue, so this is an attempt to use an unpopular kick off time.